src/services_models/services_constructor.jl

```diff
@@ -1,3 +1,32 @@
+function initialize_timeseries_names(
+    ::Type{<:PSY.Reserve},
+    ::Type{T},
+) where {T <: Union{RangeReserve, RampReserve}}
+    return Dict{Type{<:TimeSeriesParameter}, String}(
+        RequirementTimeSeriesParameter => "requirement",
+    )
+end
+
+function initialize_timeseries_names(
```

Comment:
do we need these?

Reply:
I think its only used for the ORDC type reserve, I had to add this since there is no default implementation `initialize_timeseries_names`